src/modules/fancyzones/lib/ZoneSet.cpp

```diff
@@ -284,6 +294,7 @@ ZoneSet::MoveWindowIntoZoneByDirection(HWND window, HWND windowZone, DWORD vkCod
         }
         newZone->AddWindowToZone(window, windowZone, true);
     }
+    return true;
```

Comment:
You're returning `true` even when `newZone` is null, is this intended?

Reply:
Returning true here is to mimic previous behavior, but You’re right, we should probably return true only if AddWindowToZone succeeds, mostly to avoid swallowing pressed key, and passing it on to other listeners in event chain.